pwd-service/src/main/java/org/pwd/interfaces/mailgun/MailgunClient.java

```diff
@@ -44,12 +46,27 @@ public MailgunClient(@Value("${pwd.mailbox}") String mailbox,
         headers.setContentType(MediaType.APPLICATION_FORM_URLENCODED);
     }
 
-    public boolean sendEmail(EmailMessage emailMessage) {
+    public boolean sendEmail(PlainTextEmailMessage plainTextEmailMessage) {
```

Comment:
ja to widze tak:

public boolean sendEmail(PlainTextEmailMessage plainTextEmailMessage) {
        MultiValueMap<String, String> formData = createFormData(plainTextEmailMessage, formData);
        formData.add("text", plainTextEmailMessage.getText());
        return sendEmail(formData);
    }

    public boolean sendEmail(HtmlEmailMessage htmlEmailMessage) {
        MultiValueMap<String, String> formData = createFormData(htmlEmailMessage);
        formData.add("html", htmlEmailMessage.getHtml());
        return sendEmail(formData);
    }

    private MultiValueMap<String, String> createFormData(EmailMessage emailMessage) {
        LinkedMultiValueMap<String, String> formData = new LinkedMultiValueMap<>();
        formData.add("from", emailMessage.getFrom());
        formData.add("to", emailMessage.getTo());
        formData.add("subject", emailMessage.getSubject());
        return formData;
    }